share/lib/common.sh

```diff
@@ -168,6 +168,7 @@ if [ -f "${RUDDER_JSON}" ]; then
   AGENT_RUN_INTERVAL=$(rudder_json_value 'AGENT_RUN_INTERVAL')
   RUDDER_NODE_CONFIG_ID=$(rudder_json_value 'RUDDER_NODE_CONFIG_ID')
   RUDDER_SYSLOG_PROTOCOL=$(rudder_json_value 'RUDDER_SYSLOG_PROTOCOL')
+  RUDDER_VERIFY_CERTIFICATES=$(rudder_json_value 'RUDDER_VERIFY_CERTIFICATES')
```

Comment:
does it have a defaut value ?

Reply:
No but we compare it to true so it should work